Libplanet.Tests/Libplanet.Tests.csproj

```diff
@@ -1,4 +1,4 @@
-<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
is there BOM? (or removed?)

Reply:
It's like accidentally inserted.  I'll remove it.